app/views/projects/samples/show.html.erb

```diff
@@ -51,19 +51,21 @@
         },
```

Comment:
One more thing, the checkboxes also should not display for non-owner

![image](https://github.com/phac-nml/irida-next/assets/25466211/e5132f22-0323-4def-b353-8ab6815c835e)

Reply:
As discussed on Teams, because maintainers will still need the checkboxes to concatenate, I've hidden the checkboxes for users below maintainer level using the same policy as concatenation `project update?` in 8c4be64cd61a4a7fae8b642efa2a2c6c75d0ea7f

Reply:
As mentioned in the other comment we should use the `update_sample?` policy method